src/api.graphql

```diff
@@ -445,6 +422,16 @@ query MigrateMonsterCollection($publicKey: String!, $avatarAddress: Address) {
   }
 }
 
-query CheckActivationKey($activationKey: String!) {
-  activated(invitationCode: $activationKey)
+query CheckContracted($agentAddress: Address!){
+  stateQuery{
+    contracted(agentAddress: $agentAddress){
+			contracted
+    }
+  }
+}
+
+query approvePledge($publicKey: String!){
+	actionTxQuery(publicKey: $publicKey){
+    approvePledge(patronAddress: "0x441Ad1fD834BaB75c6E5859e5Bf736c97Ffa6cc5")
```

Comment:
I think that maintaining this address as constant or configurable might be better than here. (it's the place hard to guess.)